model/metric/event.go

```diff
@@ -45,7 +45,7 @@ var (
 	processorEntry  = common.MapStr{"name": processorName, "event": docType}
 )
 
-var cachedModelSchema = validation.CreateSchema(schema.ModelSchema, processorName)
+var cachedModelSchema = validation.CreateSchema(schema.ModelSchema, "metricset")
```

Comment:
I see what you mean. I wrote a comment on the PR regarding this, but it wasn't very clear: https://github.com/elastic/apm-server/pull/1359#issuecomment-419043572

What i meant there is that in v1, it's called `metrics`. There are some places in the code that is only used by v1 which  needs to still be named "metrics", because otherwise the v1 API changes. Additionally, there are several places that is shared between v1 and v2 and where we have to choose between making it right for v1 and making it right for v2. Then there are the places that only affect v2 and those were the ones i chose to change in that PR. I'll go ahead and change it to `metricset` everywhere that doesn't affect v1 directly to minimize confusion.